cruise-control-core/src/test/java/com/linkedin/cruisecontrol/monitor/sampling/aggregator/MetricSampleAggregatorTest.java

```diff
@@ -160,6 +160,25 @@ public void testAvailableWindows() {
     }
   }
 
+  @Test
+  public void testAddSamplesWithLargeInterval() {
```

Comment:
Would it be possible to add comments to clarify what we test here? -- I believe we had an offline discussion, but I haven't left a feedback here about it.

Reply:
Ah, sorry I forget that...added the explanation fo the new test.